src/wymeditor/plugins/table/jquery.wymeditor.table.js

```diff
@@ -138,22 +138,38 @@ TableEditor.prototype.bindEvents = function () {
     jQuery(wym._box).find(
         tableEditor._options.sAddRowButtonSelector
     ).click(function () {
-        return tableEditor.addRow(wym.selectedContainer());
+        var cont = wym.selectedContainer();
+        if (cont) {
+            tableEditor.addRow(cont);
+        }
+        return false;
     });
     jQuery(wym._box).find(
         tableEditor._options.sRemoveRowButtonSelector
     ).click(function () {
-        return tableEditor.removeRow(wym.selectedContainer());
+        var cont = wym.selectedContainer();
```

Comment:
It returns `false`.

`selectedContainer` is not the exact most correct thing to use in, row removal, for one.

Because, for a selection like this one:
![screenshot_2015-11-27_18-11-50](https://cloud.githubusercontent.com/assets/635591/11445146/74f000a0-9532-11e5-9aa8-1d3b7f472491.png)
It returns `false`.

So, in `master`, this throws and in this branch it aborts row removal.

So, if you'd like, you can report this and I'll have to come up with some abstraction for such a selection, like `getSelectionFirstCommonAncestor` or something.

In any case, this is a pareto improvement, because aborting is better than throwing.